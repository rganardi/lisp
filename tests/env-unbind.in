(define 0 (quote zero))

(define 1 (quote one))
(define id (lambda (x) x))
(define list (lambda x x))
,p

(undef nil)
,p

(undef list)
,p
