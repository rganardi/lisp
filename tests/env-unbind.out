parse_eval: unspecified
parse_eval: unspecified
parse_eval: unspecified
parse_eval: unspecified
printing environment
list	->	(lambda x x)
id	->	(lambda (x) x)
1	->	one
0	->	zero
nil	->	
parse_eval: unspecified
parse_eval: unspecified
printing environment
list	->	(lambda x x)
id	->	(lambda (x) x)
1	->	one
0	->	zero
parse_eval: unspecified
parse_eval: unspecified
printing environment
id	->	(lambda (x) x)
1	->	one
0	->	zero
parse_eval: unspecified
